it/controllers/propertyLinking/ChooseEvidenceControllerISpec.scala

```diff
@@ -0,0 +1,1643 @@
+package controllers.propertyLinking
+
+import base.ISpecBase
+import binders.propertylinks.ClaimPropertyReturnToPage
+import com.github.tomakehurst.wiremock.client.WireMock._
+import models.{ClientDetails, EvidenceType, Lease, License, LinkingSession, NoLeaseOrLicense, Occupier, Owner, OwnerOccupier, PropertyOccupancy, PropertyOwnership, PropertyRelationship}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.http.HeaderNames
+import play.api.http.Status._
+import play.api.libs.json.Json
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import repositories.PropertyLinkingSessionRepository
+import uk.gov.hmrc.http.{HeaderCarrier, SessionId}
+
+import java.time.LocalDate
+import java.util.UUID
+
+class ChooseEvidenceControllerISpec extends ISpecBase {
+
+  val testSessionId = s"stubbed-${UUID.randomUUID}"
+
+  lazy val mockPropertyLinkingSessionRepository: PropertyLinkingSessionRepository = app.injector.instanceOf[PropertyLinkingSessionRepository]
+  implicit val hc: HeaderCarrier = HeaderCarrier(sessionId = Some(SessionId(testSessionId)))
+
+  val titleText = "Do you have a business rates bill for this property? - Valuation Office Agency - GOV.UK"
+  val occupierTitleText = "Do you have a lease or a licence to occupy for this property? - Valuation Office Agency - GOV.UK"
+  val clientTitleText = "Do you have your client's business rates bill for this property? - Valuation Office Agency - GOV.UK"
```

Comment:
theres a few quotes that can be cleaned up as part of these changes, seems to be a few "client''s" in the messages file

Reply:
Just updated this 👍